### PR TITLE
0.8.0-rc0 release updates

### PR DESCRIPTION
**Pull request by swcurran**
Updates necessary to create a 0.8.0-rc0 release. For this release, the CHANGELOG.md has been update to move the categorized list of Pull Requests from the "0.1.0-rc1" to under this release, and then the PRs since 1.0.0-rc1 were added.

My proposal is that after we release 0.8.0 (final), we follow up with a 0.1.0-rc2 release that is the same as 0.8.0 so that those tracking to the 1.0.0 release can use that. As needed, we can do the same for 0.8.1 / 0.9.0 if needed, but ideally we will get 1.0.0 out the door before those are needed.

@WadeBarnes @ianco @dbluhm -- does that make sense?

Also, please review the note I put into the changelog about the container images (do we have link to put into that paragraph), and if what I state is accurate.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,109 +1,167 @@
...
+
+0.8.0 is a breaking change that contains all updates since release 0.7.5. It
+extends the previously tagged `1.0.0-rc1` release because it is not clear when
+that release will be finalized. Many of the PRs in this release were previously
+included in the `1.0.0-rc1` release. The categorized list of PRs separates those
+that are new from those in the `1.0.0-rc1` release candidate.
+
+With this release, a new automated process publishes container
+images in the Hyperledger container image repository for this version of ACA-Py
+based on both Python 3.6 and 3.9. We recommend using Python 3.9 with ACA-Py.
```
Comment (WadeBarnes) [inline CHANGELOG.md:13]: Additional information can be found here; [Container Images and Github Actions](https://github.com/hyperledger/aries-cloudagent-python/blob/main/ContainerImagesAndGithubActions.md)

The new images will be published here; https://github.com/orgs/hyperledger/packages?repo_name=aries-cloudagent-python.  There will be a link to the packages on the repository's main page under **Packages** providing a more direct link once the first images are published.

The images are based on `python slim-bullseye` images, and are built to support `linux/386` (`x86`), `linux/amd64` (`x64`), and `linux/arm64`.  There are two flavors of image built for each python version.  One containing the indy/aries shared libraries only, and one containing the indy/aries shared libraries and the indy-sdk.  The images containing the indy-sdk are labeled `indy`.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]

**Comment by WadeBarnes** [comment]
> My proposal is that after we release 0.8.0 (final), we follow up with a 0.1.0-rc2 release that is the same as 0.8.0 so that those tracking to the 1.0.0 release can use that. As needed, we can do the same for 0.8.1 / 0.9.0 if needed, but ideally we will get 1.0.0 out the door before those are needed.
> 
> @WadeBarnes @ianco @dbluhm -- does that make sense?

Yes, that makes sense.

> Also, please review the note I put into the changelog about the container images (do we have link to put into that paragraph), and if what I state is accurate.

Reviewed, feedback above.

**Comment by ianco** [comment]
> > My proposal is that after we release 0.8.0 (final), we follow up with a 0.1.0-rc2 release that is the same as 0.8.0 so that those tracking to the 1.0.0 release can use that. As needed, we can do the same for 0.8.1 / 0.9.0 if needed, but ideally we will get 1.0.0 out the door before those are needed.
> > @WadeBarnes @ianco @dbluhm -- does that make sense?
> 
> Yes, that makes sense.

Makes sense to me too

**Comment by swcurran** [comment]
Updates made @WadeBarnes -- please review.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,109 +1,197 @@
...
+
+With this release, a new automated process publishes container images in the
+Hyperledger container image repository. New images for the release are
+automatically published by the GitHubAction Workflows: [publish.yml] and
+[publish-indy.yml]. The actions are triggered when a release is tagged, so no
+manual action is needed. The images are published in the [Hyperledger Package
+Repository under aries-cloudagent-python] and a link to the packages added to
+the repositories main page (under "Packages"). Additional information about the
+container image publication process can be found in the document [Container
+Images and Github Actions].
```
Comment (WadeBarnes) [inline CHANGELOG.md:19]:
```suggestion
container image publication process can be found in the document [Container Images and Github Actions](https://github.com/hyperledger/aries-cloudagent-python/blob/main/ContainerImagesAndGithubActions.md).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,109 +1,197 @@
...
+have both local resolvers for some DID Methods and a call out to an external
+universal resolver for other DID Methods. Another significant new capability is
+full support for Hyperledger Indy transaction endorsement for Authors and
+Endorsers. A new repo
 [aries-endorser-service](https://github.com/hyperledger/aries-endorser-service)
 has been created that is a pre-configured instance of ACA-Py for use as an
-Endorser service. While some work has been done on moving the default Python
-version beyond 3.6, more work is still to be done on that before the final
-v1.0.0 release.
+Endorser service.
```
Comment (WadeBarnes) [inline CHANGELOG.md:30]: Did you mean the separate the paragraphs describing the new container images with this paragraph?  It breaks the flow a bit.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
A couple small items

File: CHANGELOG.md (hunk 2 above)
Comment (WadeBarnes) [reply CHANGELOG.md:19]: That didn't format well.  I was trying to point out the missing link for `[Container Images and Github Actions]` and recommend it be changed to `[Container Images and Github Actions](https://github.com/hyperledger/aries-cloudagent-python/blob/main/ContainerImagesAndGithubActions.md)`

Comment (swcurran) [reply CHANGELOG.md]: Fixed — I missed copying over the link targets from the publishing doc.

File: CHANGELOG.md (hunk 3 above)
Comment (swcurran) [reply CHANGELOG.md:20]: I did mean to do that, but it does make it look terrible. The first part was intended to be a “new capability”, and the second part some extra detail.  I combined them as you will see.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,109 +1,203 @@
...
+
+### Container Publishing Updated
+
+With this release, a new automated process publishes container images in the
+Hyperledger container image repository. New images for the release are
+automatically published by the GitHubAction Workflows: [publish.yml] and
```
Comment (WadeBarnes) [inline CHANGELOG.md:26]:
```suggestion
automatically published by the GitHubAction Workflows: [publish.yml](https://github.com/hyperledger/aries-cloudagent-python/blob/main/.github/workflows/publish.yml) and
```

Add missing link.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,109 +1,203 @@
...
+[publish-indy.yml]. The actions are triggered when a release is tagged, so no
```
Comment (WadeBarnes) [inline CHANGELOG.md:27]:
```suggestion
[publish-indy.yml](https://github.com/hyperledger/aries-cloudagent-python/blob/main/.github/workflows/publish-indy.yml). The actions are triggered when a release is tagged, so no
```

Add missing link.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,109 +1,203 @@
...
+manual action is needed. The images are published in the [Hyperledger Package
+Repository under aries-cloudagent-python] and a link to the packages added to
+the repositories main page (under "Packages"). Additional information about the
+container image publication process can be found in the document [Container
+Images and Github Actions].
```
Comment (WadeBarnes) [inline CHANGELOG.md:32]:
```suggestion
container image publication process can be found in the document [Container Images and Github Actions](https://github.com/hyperledger/aries-cloudagent-python/blob/main/ContainerImagesAndGithubActions.md).
```

Add missing link.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
Way better!  Just some missing links to add.

File: CHANGELOG.md (hunk 4 above)
Comment (swcurran) [reply CHANGELOG.md:26]: They aren’t missing — they are defined below, Lines 54-57.

File: CHANGELOG.md (hunk 6 above)
Comment (swcurran) [reply CHANGELOG.md:32]: Not missing.

**Comment by swcurran** [comment]
@WadeBarnes — please check the rendered version to see the links are there.

**Comment by WadeBarnes** [comment]
> @WadeBarnes — please check the rendered version to see the links are there.

Sorry, my oversight.

**Review by WadeBarnes — DISMISSED** [review]

**Comment by swcurran** [comment]
@WadeBarnes -- please approve again -- added last PR merged (#2112). 

Note -- I turned off the new queuing feature as it seem to be hanging.  It was used on on 2112, but the required checks (that had already passed) were re-run and didn't complete in time.  We'll see about reactivating later...

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**